operator/docs/forwarding_logs_to_gateway.md

```diff
@@ -6,11 +6,119 @@ __Please read the [hacking guide](./hack_loki_operator.md) before proceeding wit
 
 _Note: While this document will only give instructions for two methods of log forwarding into the gateway, the examples given in the Promtail and Fluentd sections can be extrapolated to other log forwarders._
 
-## Openshift Logging
-
-Although there is a way to [forward logs to an external Loki instance](https://docs.openshift.com/container-platform/4.9/logging/cluster-logging-external.html#cluster-logging-collector-log-forward-loki_cluster-logging-external), [Openshift Logging](https://github.com/openshift/cluster-logging-operator) does not currently have support to send logs through the Lokistack Gateway.
-
-Support will be added in the near future.
+## OpenShift Logging
+
+[OpenShift Logging](https://github.com/openshift/cluster-logging-operator) supports [forwarding logs to an external Loki instance](https://docs.openshift.com/container-platform/4.9/logging/cluster-logging-external.html#cluster-logging-collector-log-forward-loki_cluster-logging-external). This can also be used to forward logs to LokiStack gateway.
+
+* Deploy the Loki Operator and an `lokistack` instance with the [gateway flag enabled](./hack_loki_operator.md#hacking-on-loki-operator-on-openshift).
+
+* Deploy the OpenShift Logging Operator from the Operator Hub or using the following command locally:
+
+    ```console
+    make deploy-image deploy-catalog install
+    ```
```

Comment:
If we provide make targets for manual installation of the openshift/cluster-logging-operator, we should also provide a link to that repository.